File: 프로그래머스/lv1/86491. 최소직사각형/최소직사각형.js
function solution(sizes) {
    var minStack = [];
    var maxStack = [];
    for (size of sizes)
    {
        maxStack.push(Math.min(...size));
        minStack.push(Math.max(...size));  
    }
   return (Math.max(...minStack)*Math.max(...maxStack));
}

File: 프로그래머스/lv1/86491. 최소직사각형/최소직사각형.py
def solution(sizes):
    answer = 0
    w = 0
    h = 0
    sizes.sort(reverse=True)
    for i,j in sizes:
        if max(i,j)> w : w = max(i,j)
        if min(i,j)> h : h = min(i,j)    
    return w*h